src/mongocxx/collection.hpp

```diff
@@ -477,6 +477,16 @@ class MONGOCXX_API collection {
     stdx::string_view name() const noexcept;
 
     ///
+    /// Rename this collection.
+    ///
+    /// @param new_name The new name to assign to the collection.
+    /// @param drop_target_before_rename Whether to overwrite any
+    ///   existing collections called new_name. The default is false.
+    /// @throws exception::operation if the operation fails.
+    void name(stdx::string_view new_name, bool drop_target_before_rename = false);
```

Comment:
Just some style things, please change your doc comment indentation so it reads like:
```
/// @param new_name
///   The new name to assign to the collection.
```
Also, please add an empty `///` line between the `@param`'s and the `@throws`, and one at the end before the function declaration.